spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaDashboardProperties.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.netflix.eureka.server;
 
-import lombok.Data;
-
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
As a point, we try not to rely on libraries like commons-lang and guava. See 31d19fd51ad7bfdc035c09928cd677669639e764